src/instana/instrumentation/pika.py

```diff
@@ -2,167 +2,244 @@
 # (c) Copyright IBM Corp. 2021
 # (c) Copyright Instana Inc. 2021
 
+import types
+from typing import TYPE_CHECKING, Any, Callable, Dict, Generator, Optional, Tuple, Union
 
-
+import pika.connection
```

Comment:
Please move this inside the try block below to avoid import errors.

Reply:
Done